src/functionalTest/java/uk/gov/hmcts/reform/orgrolemapping/befta/OrgRoleMappingAmTestAutomationAdapter.java

```diff
@@ -37,6 +38,18 @@ public Object calculateCustomValue(BackEndFunctionalTestScenarioContext scenario
                 }
                 logger.info("The nap is complete.");
                 return null;
+            case ("generateServiceBusToken"):
+                try {
+                    return FunctionalTestUtils.getSaSToken("sb://"
+                                    + EnvironmentVariableUtils.getRequiredVariable("AMQP_HOST"),
+                            "SendAndListenSharedAccessKey",
+                            EnvironmentVariableUtils.getRequiredVariable("AMQP_SHARED_ACCESS_KEY_VALUE"));
+                } catch (Exception e) {
+                    logger.warn(e.getMessage());
+                }
+                return null;
+            /*case("getUserIdFromFile"):
+                return FunctionalTestUtils.getUserIdFromFile("SingleUserId.json");*/
```

Comment:
Do we need to keep this commented line of code?

Reply:
Aashish working on this Adaptor class.